auth-foundation/src/main/java/com/okta/authfoundation/credential/storage/TokenEntity.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2024-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.authfoundation.credential.storage
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import androidx.room.TypeConverters
+import com.okta.authfoundation.credential.storage.typeconverters.JsonObjectTypeConverter
+import com.okta.authfoundation.credential.storage.typeconverters.StringStringMapTypeConverter
+import kotlinx.serialization.json.JsonObject
+
+@Entity
+@TypeConverters(
+    StringStringMapTypeConverter::class,
+    JsonObjectTypeConverter::class
+)
+internal data class TokenEntity(
+    @PrimaryKey
+    val id: String,
+    val encryptedToken: ByteArray,
+    val tags: Map<String, String>,
+    val payloadData: JsonObject,
+    val keyAlias: String,
+    val tokenEncryptionType: EncryptionType,
+    val isDefault: Boolean,
+    val encryptionExtras: Map<String, String>
+) {
+    internal enum class EncryptionType {
+        NON_BIO,
```

Comment:
How about `default` or `standard`? `non_bio` is too specific in my mind, and won't make much sense if other encryption types other than bio and pin are added in the future.